resources/lib/uzg.py

```diff
@@ -9,16 +9,24 @@
     Uitzendinggemist (NPO) = Made by Bas Magre (Opvolger)    
 
 '''
+import xbmc
```

Comment:
Deze import is hier niet nodig, ik probeer alleen xbmc aan te spreken in de addon.py zodat ik de rest los kan aanroepen om te testen. Dat gaat een stuk fijner met debuggen